### PR TITLE
[spec] Clean up entropy budget section

### PR DESCRIPTION
**Pull request by pythagoraskitty**
We clean up the [Entropy Budgets](https://wicg.github.io/shared-storage/#budgets) section, addressing #147, #149, #150, and #184 . A followup will address #183 .

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/shared-storage/pull/185.html" title="Last updated on Sep 10, 2024, 7:21 PM UTC (eb72cba)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/shared-storage/185/8352561...eb72cba.html" title="Last updated on Sep 10, 2024, 7:21 PM UTC (eb72cba)">Diff</a>

### CODE REVIEW

**Comment by pythagoraskitty** [comment]
@xyaoinum PTAL, thanks!

File: spec.bs [hunk 1]
```diff
@@ -825,23 +824,44 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
...
 
   Because [=bits of entropy=] can leak via {{SharedStorageWorklet/selectURL()}}, the [=user agent=] will need to maintain budgets to limit these leaks.
 
-  ### Navigation Entropy Budget ### {#nav-budget}
+  On a call to {{SharedStorageWorklet/selectURL()}}, when any of these budgets are exhausted, the [=default selectURL index=] will be used to determine which URL to select.
+
+  <div algorithm>
+    To <dfn>get the default selectURL index</dfn>, given {{sequence}}<{{USVString}}> |urls|, run the following steps:
+
+    1. Return an {{unsigned long}} from the [=the exclusive range|range=] from 0 to |urls|’s [=list/size=], exclusive in [=implementation-defined=] way so that the returned index is independent from the registered operation class's "`run`" method..
```
Comment (xyaoinum) [inline spec.bs:837]: nit: Redundant period at the end

File: spec.bs [hunk 2]
```diff
@@ -825,23 +824,44 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
...
+  </div>
+
+  The <dfn>default selectURL index</dfn> is the index obtained by running [=get the default selectURL index=], given {{sequence}}<{{USVString}}> |urls|.
```
Comment (xyaoinum) [inline spec.bs:840]: Now that the clean-up touches this section, can we just define it to be 0 rather than making it implementation-defined?

**Review by xyaoinum — APPROVED** [review]
LGTM % comments

File: spec.bs (hunk 2 above)
Comment (pythagoraskitty) [reply spec.bs]: Done.

File: spec.bs (hunk 1 above)
Comment (pythagoraskitty) [reply spec.bs]: Fixed.

**Pull request merged**